.circleci/config.yml

```diff
@@ -46,7 +46,7 @@ jobs:
           key: v3-npm-deps-{{ arch }}-{{ checksum "yarn.lock" }}-{{ checksum ".yarnclean" }}
           paths:
             - node_modules
-      - run: npx chroma publish --build-script-name='storybook:build'
+      - run: npx chromatic publish --build-script-name='storybook:build'
```

Comment:
Are we supposed to run this using npx? I thought this method was only recommended for projects that don't have the package listed in their dependencies?

Reply:
npx will check the local path and installed modules before downloading it, (as `npm run` only looks at scripts, it doesn't pick up binarys like `yarn run`).

I've changed this to `yarn run chromatic...` so we're consistently using yarn

Reply:
This was a copy+paste from their original docs. npx will check the local path and installed modules before downloading it, (as `npm run` only looks at scripts, it doesn't pick up binarys like `yarn run`) it's not just for if you've not got something in your deps.

I've changed this to `yarn run chromatic...` which is functionally the same as we install their package except now we're consistently using yarn